### ISSUE TITLE
Highlight text

### ISSUE THREAD

**Issue by ayoubrajabi** [payam-zahedi/toastification#61]
Hi.
I need to add highlight text to the title and description.
for example like this:

<img width="296" alt="Screenshot 2023-10-23 at 12 04 08" src="https://github.com/payam-zahedi/toastification/assets/73716390/5349103e-56de-4cfe-b07f-b18096bff6d3">

I can work on it if you like.

**Comment by payam-zahedi** [comment]
Do you need RichText instead of simple text?

**Comment by ayoubrajabi** [comment]
yes. I need RichText.

**Comment by ayoubrajabi** [comment]
@payam-zahedi 
Please let me know what are you planning for this idea. 
If you want to add this feature, I'm interested in working on this feature.

**Comment by callmephil** [comment]
> Do you need RichText instead of simple text?

What about passing an optional Widget instead of Strings for title and subtitle? That way we can do whatever we want with toast!

```dart
toastification.show(
  title: Text('Hello, World!'),
  description:  RichText(text: TextSpan(...)),
   ...
 }
  ```

**Comment by RoyalCoder88** [comment]
@callmephil I think as well this will be useful, but I think you can achieve this with: ```toastification.showCustom```

**Comment by payam-zahedi** [comment]
@callmephil 

You right 
sending a widget instead of string could be a solution, but I want to think about it
I want to keep the API clean.

**Comment by callmephil** [comment]
> @callmephil I think as well this will be useful, but I think you can achieve this with: ```toastification.showCustom```

Providing widgets instead of Strings just makes it friendly to code and maintain.

@payam-zahedi I can take a look if you want

**Comment by payam-zahedi** [comment]
Thank you @callmephil for proposing it 

Feel free to start work on it

**Comment by payam-zahedi** [comment]
We fixed this issue in this PR #67 

So you will be able to use it the next version
